### DISCUSSION TITLE
graphql/codegen/docs/guides/react-query

### DISCUSSION THREAD

**Discussion by giscus** [dotansimha/graphql-code-generator#10078] [Docs Discussions]
# graphql/codegen/docs/guides/react-query

Execute GraphQL queries type-safe in React Query with GraphQL Code Generator.

https://the-guild.dev/graphql/codegen/docs/guides/react-query

**Comment by michaeldev5** [comment]
In the fetcher the data field is not unwrapped and because of that types do not match.
